test/ote/network_tools.go

```
@@ -0,0 +1,280 @@
+	})
+
+	g.It("[JIRA:Networking][OTP] 67625-67648-Check ovnkube-trace pod2pod and pod2hostnetworkpod traffic", func() {
+		nodeList, err := getReadySchedulableNodes(ctx, clientset)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if len(nodeList.Items) < 2 {
+			g.Skip("Not enough nodes available, need at least 2")
+		}
+		workerNode1 := nodeList.Items[0].Name
+		workerNode2 := nodeList.Items[1].Name
+		tmpPath := "/tmp/ocp-67625-67648"
+		defer os.RemoveAll(tmpPath)
```

golangci-lint (2.12.2)

[error] Error return value of `os.RemoveAll` is not checked

(errcheck)